libs/vre/shared/app-common/src/lib/resource.util.ts

```diff
@@ -0,0 +1,13 @@
+import { PermissionUtil, ReadResource, ReadValue } from '@dasch-swiss/dsp-js';
+
+export class ResourceUtil {
+  public static userCanEdit(instance: ReadResource | ReadValue) {
+    const permissions = PermissionUtil.allUserPermissions(instance.userHasPermission as 'RV' | 'V' | 'M' | 'D' | 'CR');
+    return permissions.indexOf(PermissionUtil.Permissions.M) !== -1;
```

Comment:
could you replace it with: return persmissions.includes(PermissionUtil.Permissions.M);

It makes more sense IMO

Reply:
nicer indeed.